security/azure/pg_log_retention/pg_log_retention.pt

```diff
@@ -0,0 +1,198 @@
+name "Azure Ensure PostgreSQL Servers Sufficient Log Retention"
+rs_pt_ver 20180301
+type "policy"
+short_description "Report if any PostgreSQL server instances do not have log retention configured for more than 3 days. \n See the [README](https://github.com/flexera/policy_templates/tree/master/security/azure/pg_log_retention) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
```

Comment:
Regarding the 2nd link - replace RightScale docs link with Flexera Automation docs link